aries-site/src/pages/foundation/voice-and-tone.mdx

```diff
@@ -170,29 +170,38 @@ These concrete rules structure our communications.
 
 ## Capitalization
 
-The general rule is that things that are sentences should follow sentence capitalization rules. And, things that are titles/headings/labels should follow title capitalization rules.
-
-Sentences follow sentence capitalization rules:
-
+There are four types of capitalization. 
+1. Sentence case, where the first letter is a capital, the second and all subsequent words begin with lowercase letters.
+   (example: Storage volume) 
+2. Title case, where most words are capitalized except for "connector" words such as "the," "an," "or," etc.
+   (example: "The Benefits of Cloud Storage Explained")
+3. Camel case, where there are capital letters inside the word, almost always for branding purposes.
+    (example: GreenLake)
+4. All lower case, where every single word is lowercase 
+    (example: "lower case exudes an approachable and casual vibe")
+
+At HPE, almost all text should be written in sentence case. 
+
+Sentence case:
```

Comment:
```suggestion
Sentence case should be used for:
```
and similar pattern for the rest

Reply:
Does this sound good?

"Sentence case should be applied on:"